package.json

```diff
@@ -2,5 +2,8 @@
   "devDependencies": {
     "jsdoc-to-markdown": "^6.0.1",
     "limax": "^2.1.0"
+  },
+  "dependencies": {
+    "on-change": "^2.2.2"

```

Comment:
Unless *all* providers use this (I think maybe mongo doesn't?) this should be in each provider's package.json. Either that or add the feature to mongo, which I didn't think of adding.

Reply:
Yea that was my bad, I ran yarn add in the parent dir by accident